src/Aspire.Hosting/Dcp/DcpExecutor.cs

```diff
@@ -1696,8 +1696,6 @@ public async Task StartResourceAsync(string resourceName, CancellationToken canc
 
         try
         {
-            await _executorEvents.PublishAsync(new OnResourceStartingContext(cancellationToken, resourceType, matchingResource.ModelResource, matchingResource.DcpResource.Metadata.Name)).ConfigureAwait(false);
```

Comment:
This used to get raised for ExecutableReplicaSets before. Does it still get raised with this change?

Reply:
ExecutableReplicaSets is legacy code. It should be removed. But I'll do that in its own PR.